.lsan.supp

```diff
@@ -19,6 +19,7 @@ leak:TKernel
 leak:libtbb
 
 # Potential mesa/VTK leak with incomplete callstack
-# forces us to hide all leaks from the libf3dSDKTests
+# forces us to hide all leaks from the tests using a render window
 # https://gitlab.kitware.com/vtk/vtk/-/issues/18504
 leak:libf3dSDKTests
+leak:VTKExtensionsRenderingTests
```

Comment:
I suppose this is what you refered to when saying there is something wrong in VTK.
Do you plan to investigate it ?

Reply:
I investigated and failed to find anything